features/storage/kvstore/securestore/mbed_lib.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "SecureStore",
+    "macros": ["MBEDTLS_CIPHER_MODE_CTR", "MBEDTLS_CMAC_C"],
+    "config": {
+    }
+}
```

Comment:
Something strange at the end of this file

Reply:
All I need here is to have the cipher and CMAC defines set. Nothing need in the config.